runtime/vm/JVMImage.cpp

```diff
@@ -481,6 +481,24 @@ setupJVMImagePortLibrary(J9JavaVM *javaVM)
 	return jvmImagePortLibrary;
 }
 
+void
+JVMImage::storeInitialMethods(J9Method *cInitialStaticMethod, J9Method *cInitialSpecialMethod, J9Method *cInitialVirtualMethod, J9Method *cInvokePrivateMethod)
+{
+	_jvmImageHeader->cInitialStaticMethod = cInitialStaticMethod;
+	_jvmImageHeader->cInitialSpecialMethod = cInitialSpecialMethod;
+	_jvmImageHeader->cInitialVirtualMethod = cInitialVirtualMethod;
+	_jvmImageHeader->cInvokePrivateMethod = cInvokePrivateMethod;
+}
+
+void
+JVMImage::setInitialMethods(J9Method **cInitialStaticMethod, J9Method **cInitialSpecialMethod, J9Method **cInitialVirtualMethod, J9Method **cInvokePrivateMethod)
```

Comment:
I know I said we could do it this way. But this is also an option. 
```
JVMImage::setInitialMethods(J9Method *&cInitialStaticMethod, J9Method *&cInitialSpecialMethod, J9Method *&cInitialVirtualMethod, J9Method *&cInvokePrivateMethod)
```
This way you pass the pointer as reference and so you could just directly do this:
` cInitialStaticMethod = _jvmImageHeader->cInitialStaticMethod;`
Also when you call the function you dont have to pass the address
@Ershany-IBM which one do you this is cleaner.